roboy_middleware_msgs/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ find_package(ament_cmake REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(geometry_msgs REQUIRED)
+find_package(roboy_simulation_msgs REQUIRED)
```

Comment:
hm, this is not ideal

Reply:
I can move the LinkInformation msg from simulation to middleware, its only needed there